packages/react-centra-checkout/src/PaymentEmbed/index.tsx

```diff
@@ -10,6 +10,23 @@ export interface PaymentEmbedProps {
   onPaymentError?: (error: CheckoutApi.Errors) => void
 }
 
+function isRecordStringString(value: unknown): value is Record<string, string> {
+  // Check if the value is an object and not null
+  if (typeof value !== 'object' || value === null) {
+    return false
+  }
+
+  // Iterate over the keys of the object
+  for (const key in value) {
+    // Check if the key is a string and the value is a string
+    if (typeof key !== 'string' || typeof value[key as keyof typeof value] !== 'string') {
+      return false
+    }
+  }
+
+  return true
+}
+
```

Comment:
Maybe this would be worth putting in another package? @maeertin @alexanderflink

Reply:
Look at my comment [here](https://github.com/noaignite/accelerator/pull/153#discussion_r1714853602) :)